app/js/lib/foxbox.js

```diff
@@ -16,8 +16,6 @@ let _tunnelConfigured = false;
 let _local = false;
 // Whether we can connect to the box via a remote connection.
 let _remote = false;
-// A reference to the interval to get the online status.
-let _onlineInterval = null;
```

Comment:
It's not used and ESLint is complaining, if we decide to clear interval we can return it back